Azure/sonic-buildimage
# [device/celestica]: Implement Silverstone platform API [SFP]

**- What I did**
  - Add Sfp APIs base on SfpBase and SfpUtilBase 

**- How I did it**
1. Implement Sfp APIs.
3. Init Sfp object list on chassis API.

**- How to verify it**
- Create python script follow the new platform API design doc to verify it (Azure/SONiC#285.)
- Test Log : [sfp_api_silverstone_test.log](https://github.com/Azure/sonic-buildimage/files/3948496/sfp_api_silverstone_test.log)


Signed-off-by: Wirut Getbamrung wgetbumr@celestica.com
